taskcluster/ci/signing/kind.yml

```diff
@@ -22,7 +22,7 @@ job-template:
     signing-format: autograph_apk
     run-on-tasks-for: [github-push]
     treeherder:
-        job-symbol: S
+        job-symbol: BS
```

Comment:
nit: typically the s is lower case.. but I feel like maybe I'm being too picky at this point :)